autonick.lua

```diff
@@ -41,14 +69,24 @@ for line in io.lines(path) do
 end
 
 --assign names
+local count = 0
 for _,unit in ipairs(df.global.world.units.active) do
-    if (#names == 0) then break end
+    if (#names == 0) then
+            if options.quiet ~= true then
+                print("no free names left in autonick.txt.")
```

Comment:
Sorry, didn't see this earlier. Few people will know the path to this file. It would be clearer to say "no free names left in dfhack-config/autonick.txt"